api/core/ops/ops_trace_manager.py

```diff
@@ -740,10 +742,19 @@ def start_timer(self):
     def send_to_celery(self, tasks: list[TraceTask]):
         with self.flask_app.app_context():
             for task in tasks:
+                file_id = uuid4().hex
                 trace_info = task.execute()
                 task_data = {
```

Comment:
Consider using a pydantic model instead of a dict here, then you do not need a convert_datetime_to_str function.